src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  $daySize: 100px;
+  $borderSize: 1px;
+  $gapSize: 1px;
+  $calcWidth: ($daySize + $borderSize * 2 + $gapSize);
```

Comment:
You should use `box-sizing: border-box` to have general width with borders 100px